src/main/scala/cromwell/binding/package.scala

```diff
@@ -21,8 +21,9 @@ package object binding {
   type WorkflowCoercedInputs = Map[FullyQualifiedName, WdlValue]
   type WorkflowOutputs = Map[FullyQualifiedName, WdlValue]
   type FullyQualifiedName = String
+  type LocallyQualifiedName = String
```

Comment:
Since we've both grumbled about the type aliases, would now be a good time to stop using them in favor of things like case class wrappers?